app/views/pages/gdprpolicy.ejs

```diff
@@ -0,0 +1,71 @@
+<%- include("../partials/header") %>
+<h1>Privacy Policy for Articleblog</h1>
+
+<p>At Articleblog, accessible from http://ec2-34-229-166-184.compute-1.amazonaws.com:8000/, one of our main priorities is the privacy of our visitors. This Privacy Policy document contains types of information that is collected and recorded by Articleblog and how we use it.</p>
+
+<p>If you have additional questions or require more information about our Privacy Policy, do not hesitate to contact us. Our Privacy Policy was generated with the help of <a href="https://www.gdprprivacynotice.com/">GDPR Privacy Policy Generator from GDPRPrivacyNotice.com</a></p>
+
+<h2>General Data Protection Regulation (GDPR)</h2>
+<p>We are a Data Controller of your information.</p>
+
+<p>Articleblog legal basis for collecting and using the personal information described in this Privacy Policy depends on the Personal Information we collect and the specific context in which we collect the information:</p>
+<ul>
+    <li>Articleblog needs to perform a contract with you</li>
+    <li>You have given Articleblog permission to do so</li>
+    <li>Processing your personal information is in Articleblog legitimate interests</li>
+    <li>Articleblog needs to comply with the law</li>
+</ul>
+  
+<p>Articleblog will retain your personal information only for as long as is necessary for the purposes set out in this Privacy Policy. We will retain and use your information to the extent necessary to comply with our legal obligations, resolve disputes, and enforce our policies.</p>
+
+<p>If you are a resident of the European Economic Area (EEA), you have certain data protection rights. If you wish to be informed what Personal Information we hold about you and if you want it to be removed from our systems, you can do so through your account settings at any time.</p>
+<p>In certain circumstances, you have the following data protection rights:</p>
+<ul>
+    <li>The right to access, update or to delete the information we have on you.</li>
+    <li>The right of rectification.</li> 
+    <li>The right to object.</li>
+    <li>The right of restriction.</li>
+    <li>The right to data portability</li>
+    <li>The right to withdraw consent</li>
+</ul>
+
+<h2>Log Files</h2>
+
+<p>Articleblog follows a standard procedure of using log files. These files log visitors when they visit websites. All hosting companies do this and a part of hosting services' analytics. The information collected by log files include internet protocol (IP) addresses, browser type, Internet Service Provider (ISP), date and time stamp, referring/exit pages, and possibly the number of clicks. These are not linked to any information that is personally identifiable. The purpose of the information is for analyzing trends, administering the site, tracking users' movement on the website, and gathering demographic information.</p>
```

Comment:
What do we have to do with hosting companies?

Reply:
We host content and store their browser fingerprint, so we must add this part.